content/reference/stacks.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Stacks"
+icon: "fa fa-layer-group"
+description: "Stacks are a way to express the complete infrastructure needed for an environment using a standard YAML configuration"
+weight: 1
+tags:
+- yaml
+- atmos
+- configuration
+- concept
+---
+
+# Background
+
+Stacks are a central piece of SweetOps. They're a set of YAML files that follow a standard schema that enable a **fully declarative description of your various environments**. This empowers you with the ability to separate your infrastructure's environment configuration settings from the business logic behind it (provided via components).
+
+SweetOps utilizes YAML configuration for stacks because it's an easy to work with format that is nicely portable across multiple tools. The stack yaml format is natively supported today via [atmos](https://github.com/cloudposse/atmos), [the terraform-yaml-stack-config module](https://github.com/cloudposse/terraform-yaml-stack-config), and [Spacelift](https://spacelift.io) via [the terraform-spacelift-cloud-infrastructure-automation module](https://github.com/cloudposse/terraform-spacelift-cloud-infrastructure-automation).
+
+# Conventions
+
+We have a number of important conventions around stacks that are worth noting.
+
+## Stack Files
+
+Stack files can be very numerous in large cloud environments (think many dozens to hundreds of stack files). To enable proper organization of stack files, SweetOps recommends the following:
+
+* All stacks should be stored in a `stacks/` folder at the root of your infrastructure repository.
+* Global values you that you want to share across all stacks should go into a `stacks/globals.yaml` file
+  * This file name is special in that it will not be treated as a stack by any SweetOps automation tooling.
```

Comment:
I don't think the file is special. We shouldn't have any special files.

Reply:
(historically, it was special before we implemented imports)

Reply:
Yeah, referring to as special due to this logic in Spacelift component: 

```
  config_filenames   = fileset("../../../stacks", "*.yaml")
  stack_config_files = [for f in local.config_filenames : f if(replace(f, "globals", "") == f)]
```

Now that we'll be putting those "should be imported only" files into `stacks/catalogs/` then this is no longer a good point. 

I will be sure to update the documentation to reflect that.